public/achievementinspector.php

```diff
@@ -306,43 +227,41 @@ function toggleAllCodeRows() {
 <?php
 if ($gameIDSpecified) {
     view()->share('sidebar', true);
-    echo "<aside>";
-    echo "<div class='mb-2'>";
-    echo "<h3>Toolbox</h3>";
-    echo "<div class='flex flex-col gap-y-1'>";
+    echo "<aside class='flex flex-col gap-y-8'>";
 
-    if ($fullModifyOK || $partialModifyOK) {
-        echo "<a class='btn flex justify-center py-2' href='/achievementinspector.php?g=$gameID&f=$flag'>Refresh Page</a>";
-
-        if ($flag === AchievementFlag::Unofficial) {
-            if ($fullModifyOK) {
-                echo "<a class='btn w-full flex justify-center py-2' onclick='updateAchievementsProperty(\"flag\", " . AchievementFlag::OfficialCore . ")'>Promote Selected</a>";
-            }
-            echo "<a class='btn w-full flex justify-center py-2' href='/achievementinspector.php?g=$gameID'>Core Achievement Inspector</a>";
-        }
-        if ($flag === AchievementFlag::OfficialCore) {
-            if ($fullModifyOK) {
-                echo "<a class='btn w-full flex justify-center py-2' onclick='updateAchievementsProperty(\"flag\", " . AchievementFlag::Unofficial . ")'>Demote Selected</a>";
-            }
-            echo "<a class='btn w-full flex justify-center py-2' href='/achievementinspector.php?g=$gameID&f=5'>Unofficial Achievement Inspector</a>";
-        }
-
-        echo "<a class='btn w-full flex justify-center py-2' onclick='updateAchievementsProperty(\"type\", \"" . AchievementType::Progression . "\")'>Set Selected to $progressionLabel</a>";
-        echo "<a class='btn w-full flex justify-center py-2' onclick='updateAchievementsProperty(\"type\", \"" . AchievementType::WinCondition . "\")'>Set Selected to $winConditionLabel</a>";
-        echo "<a class='btn w-full flex justify-center py-2' onclick='updateAchievementsProperty(\"type\", null)'>Set Selected to No Type</a>";
-    }
-
-    if ($fullModifyOK) {
-        echo "<button class='btn w-full flex justify-center py-2' onclick='toggleAllCodeRows()'>Toggle Code Rows</button>";
+    $modificationLevel = 'none';
+    if ($partialModifyOK) {
+        $modificationLevel = 'partial';
+    } elseif ($fullModifyOK) {
+        $modificationLevel = 'full';
     }
 
-    echo "<a class='btn w-full flex justify-center py-2' href='/achievementinspector.php'>Back to List</a></p></div><br>";
+    $canHaveTypes = (
+        mb_strpos($gameTitle, "[Subset") === false
+        && mb_strpos($gameTitle, "~Test Kit~") === false
+    );
```

Comment:
This should also exclude Events

Reply:
Updated in latest to exclude games with console ID 101.